README.md

```diff
@@ -43,12 +43,19 @@ The `examples` repo has sample code for both clients. Note that most examples re
 
 ```
 # Example config 
+# relayer actions
 DEFENDER_RELAY_API_URL=
 DEFENDER_RELAY_POOL_ID=
 DEFENDER_RELAY_POOL_CLIENT_ID=
+# relay client
+DEFENDER_RELAY_CLIENT_API_URL=
```

Comment:
This is slightly wonky, but the "Relayer actions" (signing, etc.) is already on the most natural namespace for these variables `DEFENDER_RELAY_...` so I standardized on the name of the relevant class. It is a possible source of confusion though. Open to other ideas.

Reply:
Ah that's annoying

Reply:
I don't think we're breaking backward compatibility here, so my suggestion is that the one that should be different in naming is the `relayer actions` because is the only one that differs from every other client.

What makes sense to me is to call the `relayer actions` as `DEFENDER_RELAY_SIGNER_*` since it's actually a `signer` at the very end.

I'd go for that change unless anybody see a problem for backward compatiblity

Reply:
I think the question is if there's any chance of a consumer ever using these environment variables or if they are used strictly for internal testing. I think if internal testing, we can consider this non-breaking.Technically we'll break [our own test](https://github.com/OpenZeppelin/defender/blob/master/test/src/config.ts#L61), but I can fix that